### ISSUE TITLE
I11 We need some base format for contracts for use with escrow

### ISSUE THREAD

**Issue by TemperBead** [dreamcatcher-tech/dreamcatcher-tech.github.io#50]
otherwise the inCompletionQA step won't have any way to judge whether what was negotiated has been done.

Propose for the first loop through we add this at the bottom of the Idea doc.  Reasoning is that we need to get through the simple case first, and a contract at this stage is probably going to be too light to warrant it being elsewhere (this is an md and folder structure issue.)

**Comment by panick-carousel** [comment]
Ok sounds good to me.
So if I want to fund an Idea for completion, I fund using Pool.md, but if I want to fund with conditions, such as only this particular person, or whatever else, then I do it in a table at the bottom of the Idea document that links all the open offers ?

**Comment by TemperBead** [comment]
Yes, that's what i was thinking as well.  Also, partial funding - ie you don't mind who it is, but you put out a contract for acceptance by anyone based on advertised conditions.  So you the 'negotiation' doesn't necessarily need to be active.  it can be offer and acceptance.

E.g. I11 has an output for the DC Pool Page.  That could have a contract offered for just that part.  

The value of doing it this way is that unpopular or risky parts of an Idea can be incentivised differently from the easier/less risky parts.

If you agree, actions for I11 would be part of detailing the Fundable -> Escrowed state change.

Are there any gotchas on fragmenting the work/completion of an Idea this way on the code side?

**Comment by inverted-capital** [comment]
Work should not be fragmented.  Ideas are the control structure of outputs - if you only want some outputs, you need to make a new Idea.  QA cannot pass a partial Idea.
